packages/react-environment/src/webb-context/bridge/bridge-config.ts

```diff
@@ -1,6 +1,10 @@
 import { ChainId, WebbCurrencyId } from '@webb-dapp/apps/configs';
 import { BridgeCurrency } from '@webb-dapp/react-environment/webb-context/bridge/bridge-currency';
 
+// todo change to wrappedTokenAddresses (they are  governed token wrapper contract)
```

Comment:
I think there is value in leaving the tokenAddresses - for wrapAndDeposit functionality. Or some way to associate possible erc20/native tokens <-> wrappedToken.

Reply:
we can leave those to another PR just added them for referance